ci/travis/build-docker-images.py

```diff
@@ -152,21 +152,25 @@ def build_or_pull_base_images(is_docker_affected: bool) -> List[str]:
     is_stale = (
         datetime.datetime.now() - short_date) > datetime.timedelta(days=14)
 
-    if is_stale or is_docker_affected or _release_build():
+    print("Pulling images for caching")
+
+    _ = DOCKER_CLIENT.api.pull(
```

Comment:
any reason to do this assignment to `_`?

Reply:
I think because that is leftover from when I was developing this pipeline, I'll remove.